Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -0,0 +1,199 @@
+
+
+def get_all_installed_packs(client: demisto_client):
+    """
+
+    Args:
+        client (demisto_client): The client to connect to.
+
+    Returns:
+        list of installed python
+    """
+    try:
+        logging.info("Attempting to fetch all installed packs.")
+        response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                            path='/contentpacks/metadata/installed',
+                                                                            method='GET',
+                                                                            accept='application/json',
+                                                                            _request_timeout=None)
+        installed_packs_ids = []
+        if 200 <= status_code < 300:
+            installed_packs = ast.literal_eval(response_data)
+            installed_packs_ids = [pack.get('id') for pack in installed_packs]
+            logging.success('Successfully fetched all installed packs ')
+            logging.debug(f'The following packs were are installed:\n{installed_packs_ids}')
+        else:
+            result_object = ast.literal_eval(response_data)
+            message = result_object.get('message', '')
+            raise Exception(f'Failed to fetch installed packs - with status code {status_code}\n{message}')
+    except Exception as e:
+        logging.exception(f'The request to fetch installed packs has failed. Additional info: {str(e)}')
+
+    finally:
+        if 'Base' in installed_packs_ids:
+            installed_packs_ids.remove('Base')
+        return installed_packs_ids
+
+
+def uninstall_packs(client: demisto_client, pack_ids: list):
+    """
+
+    Args:
+        client (demisto_client): The client to connect to.
+        pack_ids: packs ids to uninstall
+
+    Returns:
+        True if uninstalling succeeded False otherwise.
+
+    """
+    body = {"IDs": pack_ids}
+    try:
+        logging.info("Attempting to uninstall all installed packs.")
+        response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                            path='/contentpacks/installed/delete',
+                                                                            method='POST',
+                                                                            body=body,
+                                                                            accept='application/json',
+                                                                            _request_timeout=None)
+        if 200 <= status_code < 300:
+            logging.success('Successfully uninstalled installed packs ')
+        else:
+            result_object = ast.literal_eval(response_data)
+            message = result_object.get('message', '')
+            raise Exception(f'Failed to uninstall packs - with status code {status_code}\n{message}')
```

Comment:
```suggestion
            raise Exception(f'[{status_code}] {message}')
```
not very important but since the logging below handles the "prefix", this is enough